__tests__/news.test.js

```diff
@@ -411,27 +411,58 @@ describe('DELETE /api/comments/:comment_id', () => {
         .delete('/api/comments/1')
         .expect(204)
         .then((response) => {
-            const responseArray = response.res.statusMessage
-            expect(responseArray).toEqual('No Content')
+            const responseMsg = response.res.statusMessage
+            expect(responseMsg).toEqual('No Content')
         })
     });
     test('should return 400 error if invalid input data type for the comment id', () => {
         return request(app)
         .delete('/api/comments/TEXT')
         .expect(400)
         .then((response) => {
-            const responseArray = response.res.statusMessage
-            expect(responseArray).toEqual("Bad Request");
+            const responseMsg = response.res.statusMessage
+            expect(responseMsg).toEqual("Bad Request");
         })
     });
     test('should return 404 error if comment id does not exist', () => {
         return request(app)
         .delete('/api/comments/999999')
         .expect(404)
         .then((response) => {
-            const responseArray = response.res.statusMessage
-            expect(responseArray).toEqual("Not Found");
+            const responseMsg = response.res.statusMessage
+            expect(responseMsg).toEqual("Not Found");
         })
     });
     
 });
+
+describe('GET /api/users', () => {
+    test('should return with the correct properties', () => {
+        return request(app)
+        .get('/api/users')
+        .expect(200)
+        .then((response) => {
+            const usersArray = response.body.users
+            const currentUsersLength = data.userData.length
+            expect(usersArray).toHaveLength(currentUsersLength)
+            usersArray.forEach((user) => {
+                expect(Object.keys(user)).toHaveLength(3)
+                expect(user).toHaveProperty('username')
+                expect(user).toHaveProperty('name')
+                expect(user).toHaveProperty('avatar_url')
+            
+        })
+    })
+})
+    test('should return correct error when invalid data type inputted', () => {
```

Comment:
this is the same test as trying to access a non-existent path so if we already have a test for htat, we dont need this one :)